docs/client/python.md

```diff
@@ -21,35 +21,47 @@ pip install openlineage-python
 
 To install the package from source, use 
 ```
-python setup.py install
+python -m pip install .
 ```
 
 ## Configuration
 
-We recommend configuring the client with an `openlineage.yml` file that tells the client how to connect to an OpenLineage backend.
+We recommend configuring the client with an `openlineage.yml` file that contains all the
+details of how to connect to your OpenLineage backend.
 
-You can make this file available to the client three ways:
+You can make this file available to the client in three ways:
 
-1. Set an environment variable to a file path: `OPENLINEAGE_CONFIG=path/to/openlineage.yml`.
-2. Put the file in the working directory.
-3. Put the file in `$HOME/.openlineage`.
+1. Set an `OPENLINEAGE_CONFIG` environment variable to a file path: `OPENLINEAGE_CONFIG=path/to/openlineage.yml`.
+2. Place an `openlineage.yml` file in the user's current working directory.
```

Comment:
I think this needs more explanation what is current working directory. It makes use of `os.getcwd()` but in relation to what process?